wordpress-mobile/WordPress-Android
# Filter posts by author

This PR adds the ability to filter the Posts List by author with the two options shown to the user being Me and Everyone. Fixes #5884

To test:
- Go to the post list of a site
- Next to the TabLayout you can tap the icon to change whether the list is showing all posts or just your own posts.
- When you change the author filter you should see the posts shown change (if applicable)
